website/docs/components/copy/snippet/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,17 @@
+## Conformance rating
+
+<Doc::Badge @type="warning">Conditionally Conformant</Doc::Badge>
+
+When the default variant of this component is used, there should not be any WCAG conformance issues with this component.
+
+However, the truncation feature is not conformant, because a keyboard-only user has no method of accessing truncated content. Users should be aware of this when evaluating whether or not to include truncation in your copy snippet.
```

Comment:
```suggestion
However, the truncation feature is not conformant, as a keyboard-only user has no method of accessing truncated content. Users should be aware of this when evaluating whether or not to include truncation in their copy snippet.
```

Reply:
Grammarly suggestion + we should talk to consumers, so I suggest:

```suggestion
However, the truncation feature is not conformant because a keyboard-only user cannot access the truncated content. Please be aware of this when evaluating whether or not to include truncation in your copy snippet.
```